cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractAssemblyOperator.java

```diff
@@ -209,16 +209,16 @@ public final CountDownLatch reconcileAll(String trigger, String namespace) {
 
         // get ConfigMaps with kind=cluster&type=kafka (or connect, or connect-s2i) for the corresponding cluster type
         List<T> desiredResources = resourceOperator.list(namespace, Labels.EMPTY);
-        Set<String> desiredNames = desiredResources.stream().map(cm -> cm.getMetadata().getName()).collect(Collectors.toSet());
+        Set<NamespacedResourceName> desiredNames = desiredResources.stream().map(cr -> new NamespacedResourceName(cr.getMetadata().getNamespace(), cr.getMetadata().getName())).collect(Collectors.toSet());
```

Comment:
Can you put the `.map` and `.collect` on separate lines to make clear the stream operations?